CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+## 1.2.0
```

Comment:
Something that I always find very useful in changelogs is the date when that version was released (ex[1]). This also makes it clear if a version has not yet been released and we are just noting the changes accruing that *will* be in that version once we publish it.

[1] https://github.com/aws/aws-encryption-sdk-python/blob/master/CHANGELOG.rst#138----2018-11-15

Reply:
I like the minor / bugfix format in the python changelog, will update to include the date.